common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -502,7 +502,7 @@ private UTF8String copyUTF8String(int start, int end) {
    * Determines if the specified character (Unicode code point) is white space or an ISO control
    * character according to Java.
    */
-  private boolean isWhitespaceOrISOControl(int codePoint) {
+  public static boolean isWhitespaceOrISOControl(int codePoint) {
```

Comment:
Because the visibility changed, please move `isWhitespaceOrISOControl` above the constructor.